.github/workflows/publish-to-nest.land.yml

```diff
@@ -0,0 +1,25 @@
+name: "publish current release to https://nest.land"
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  publishToNestDotLand:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup repo
+        uses: actions/checkout@v2
+
+      - name: "setup" # check: https://github.com/actions/virtual-environments/issues/1777
+        uses: denolib/setup-deno@v2 
+        with:
+         deno-version: v1.4.6
+
+      - name: "check nest.land"
+        run: | 
+          deno run --allow-net --allow-read --allow-run https://deno.land/x/cicd/publish-on-nest.land.ts ${{ secrets.GITHUB_TOKEN }} ${{ secrets.NESTAPIKEY }} ${{ github.repository }}
```

Comment:
I'm a little nervous about shipping my GITHUB_TOKEN secret here.

Reply:
Hi @hayd: :) I can understand - that is good feedback for me - perhaps using a fixed version of the cicd module might be better from a security perspective:  
```ts
deno run --allow-net --allow-read --allow-run https://deno.land/x/cicd@v0.8.4/publish-on-nest.land.ts ${{ secrets.GITHUB_TOKEN }} ${{ secrets.NESTAPIKEY }} ${{ github.repository }}
```

Reply:
Hi hayd
I speak on behalf of nest.land, soon you won't have to provide your github token anymore 👍 
When nest will be in stable version, you will just have to do the following command:
```shell script
nest publish --git-tag
```
That way, deno.land and nest.land are always in sync

Reply:
Definitely interested to see that! Best of luck with nest.land.

(Sending github token is a non-starter since token permissions are too coarse.)